apps/web/pages/_document.tsx

```diff
@@ -49,7 +51,7 @@ class MyDocument extends Document<Props> {
           <meta name="msapplication-TileColor" content="#ff0000" />
           <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
           <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
-          {(process.env.NODE_ENV === "development" || process.env.VERCEL_ENV === "preview") && (
+          {(!IS_PRODUCTION || process.env.VERCEL_ENV === "preview") && (
```

Comment:
Relying more on `IS_PRODUCTION` to be better prepared for backward compatibility.